website_and_docs/content/blog/2023/whats-new-in-selenium-manager-with-selenium-4.11.0.md

```diff
@@ -0,0 +1,125 @@
+DEBUG   chrome detected at C:\Program Files\Google\Chrome\Application\chrome.exe
+DEBUG   Using shell command to find out chrome version
+DEBUG   Running command: wmic datafile where name='C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe' get Version /value
+DEBUG   Output: "\r\r\n\r\r\nVersion=115.0.5790.110\r\r\n\r\r\n\r\r\n\r"
+DEBUG   Detected browser: chrome 115.0.5790.110
+DEBUG   Reading metadata from https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json
+DEBUG   Required driver: chromedriver 115.0.5790.102
+DEBUG   Driver URL: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/115.0.5790.102/win64/chromedriver-win64.zip
+INFO    Driver path: C:\Users\boni\.cache\selenium\chromedriver\win64\115.0.5790.102\chromedriver.exe
+INFO    Browser path: C:\Program Files\Google\Chrome\Application\chrome.exe
+```
+
+In this case, the local Chrome (in Windows) is detected by Selenium Manager. Then, using its version and the CfT endpoints, the proper chromedriver version (115, in this example) is downloaded to the local cache. Finally, Selenium Manager provides two results: i) the driver path (downloaded) and ii) the browser path (local).
+
+Let's consider another example. Now we want to use Chrome beta. Therefore, we invoke Selenium Manager specifying that version label as follows (notice that the CfT beta is discovered, downloaded, and stored in the local cache):
+
+```
+> selenium-manager --browser chrome --debug --browser-version beta
+DEBUG   Checking chromedriver in PATH
+DEBUG   Running command: chromedriver --version
+DEBUG   Output: ""
+DEBUG   chromedriver not found in PATH
+DEBUG   Checking chrome in PATH
+DEBUG   Running command: where chrome
+DEBUG   Output: ""
+DEBUG   chrome not found in PATH
+DEBUG   chrome has not been discovered in the system
+DEBUG   Reading metadata from https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions-with-downloads.json
+DEBUG   Required browser: chrome 116.0.5845.49
+DEBUG   Downloading chrome 116.0.5845.49 from https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.49/win64/chrome-win64.zip
+DEBUG   chrome 116.0.5845.49 has been downloaded at C:\Users\boni\.cache\selenium\chrome\win64\116.0.5845.49\chrome.exe
+DEBUG   Reading metadata from https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json
+DEBUG   Required driver: chromedriver 116.0.5845.49
+DEBUG   Driver URL: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/116.0.5845.49/win64/chromedriver-win64.zip
+INFO    Driver path: C:\Users\boni\.cache\selenium\chromedriver\win64\116.0.5845.49\chromedriver.exe
+INFO    Browser path: C:\Users\boni\.cache\selenium\chrome\win64\116.0.5845.49\chrome.exe
+```
+
+### Troubleshooting in the bindings
+
+If you want to get the Selenium Manager debugging information when using the Selenium bindings languages, you can enable it as follows (in Java):
```

Comment:
This should be language agnostic. Link to the logging page, and link to https://www.selenium.dev/documentation/webdriver/troubleshooting/logging/#logger-filtering if people want to only see the selenium manager output.

Reply:
Done. I changed all the examples in which Java is used. Instead, descriptions and links to the doc are used.